tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php

```diff
@@ -36,10 +36,10 @@ public function testConfigureDefaultToNull()
         $fixer->configure(self::$defaultTestConfig);
         $fixer->configure(null);
 
-        $defaultConfigProperty = new \ReflectionProperty('PhpCsFixer\Fixer\ClassNotation\ClassDefinitionFixer', 'defaultConfig');
+        $defaultConfigProperty = new \ReflectionProperty('PhpCsFixer\Fixer\ClassNotation\ClassDefinitionFixer', 'defaultConfiguration');
         $defaultConfigProperty->setAccessible(true);
 
-        $this->assertAttributeSame($defaultConfigProperty->getValue(), 'config', $fixer);
+        $this->assertAttributeSame($defaultConfigProperty->getValue($fixer), 'config', $fixer);
```

Comment:
argument is obligatory when it is a instance-level property
when it is static, class-level property the parameter should not be used